drivers/sdp3x/include/sdp3x_params.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2019 Dirk Ehmen
+ *               2020 Nishchay Agrawal
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sensors
+ * @{
+ * @file
+ * @brief       Device driver params interface for the SDP3x sensor.
+ *
+ * @author      Dirk Ehmen       <ehmen@ibr.cs.tu-bs.de>
+ * @author      Nishchay Agrawal <f2016088@pilani.bits-pilani.ac.in>
+ * @}
+ */
+
+#ifndef SDP3X_PARAMS_H
+#define SDP3X_PARAMS_H
+
+#include "board.h"
+#include "sdp3x.h"
+#include "saul_reg.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    SDP3x I2C addresses
+ * @{
+ */
+#define SDP3X_ADDR1                   (0x21) /* 7 bit address */
+#define SDP3X_ADDR2                   (0x22) /* 7 bit address */
+#define SDP3X_ADDR3                   (0x23) /* 7 bit address */
+/** @} */
+
+/**
+ * @name    SDP3x Models
+ * @{
+ */
+#define SDP3X_MODEL_31                1
+#define SDP3X_MODEL_32                2
+
+#define SDP31_PRODUCT_NO_BYTE_0       0x03
+#define SDP31_PRODUCT_NO_BYTE_1       0x01
+#define SDP31_PRODUCT_NO_BYTE_2       0x01
+#define SDP31_PRODUCT_NO_BYTE_3       0x01
+/** @} */
+
+/**
+ * @name    Set default configuration parameters for the SDP3X
+ * @{
+ */
+#ifndef SDP3X_PARAM_I2C_DEV
+#define SDP3X_PARAM_I2C_DEV         I2C_DEV(0)
+#endif
+#ifndef SDP3X_PARAM_I2C_ADDR
+#define SDP3X_PARAM_I2C_ADDR        SDP3X_ADDR1
+#endif
+#ifndef SDP3X_PARAM_IRQ_PIN
+#define SDP3X_PARAM_IRQ_PIN         (GPIO_PIN(PORT_A, 8))
```

Comment:
```suggestion
#define SDP3X_PARAM_IRQ_PIN         (GPIO_PIN(0, 8))
```

`PORT_A` is not defined on all platforms.

Reply:
Yeah I was wondering that. But then `GPIO_PIN(0, 8)` may also not be defined in many platforms since you may not have pin 8

Reply:
Having 9 pins/port doesn't seem unreasonable.
And those are just the defaults, will be changed by the user - this is just about getting CI to be happy.

Reply:
Ohkay. I will just change it to a lower value then to be on the safer side ;).